RecoTracker/DeDx/python/dedxEstimators_cff.py

```diff
@@ -78,3 +80,15 @@
 doAlldEdXEstimators = cms.Sequence(doAlldEdXEstimatorsTask)
 
 fastSim.toReplaceWith(doAlldEdXEstimatorsTask, cms.Task(dedxHarmonic2, dedxPixelHarmonic2))
+
+# use only the strips for Run-3
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(dedxHitInfo,
+    lowPtTracksEstimatorParameters = cms.PSet(
+        fraction = cms.double(0),
+        exponent = cms.double(-2.0),
+        truncate = cms.bool(False)
+    ),
+    usePixelForPrescales = False
+)
+
```

Comment:
still empty new lines here.

Reply:
well I thought this is desired, otherwise github adds the red circle here... but fine, I remove it now

Reply:
no, the red circle is given if the file is not ending with a newline, but here you have 2, one empty and one at EOF.

Reply:
ah ok I see, thanks for clarifying that